napari/_qt/widgets/qt_keyboard_settings.py

```diff
@@ -293,13 +293,48 @@ def _set_table(self, layer_str: str = ''):
             self._table.setItem(0, 0, item)
 
     def _get_layer_actions(self):
+        """
+        Get all the actions that could potencially raise a keybinding conflict
+        taking into account the current selected keybinding group.
```

Comment:
Nitpick, could we follow docstring format: https://peps.python.org/pep-0257/#multi-line-docstrings
(one liner, followed by paragraphs). 

I think we could fit it in one line, maybe something like:

```suggestion
        Get all actions we want to avoid keybinding conflicts with.
        
        If current selected keybinding group is a layer, return
        the selected layer actions and viewer actions.
        If the current selected keybinding group is viewer,
        return actions from all layers.
```